base/src/main/java/com/smartdevicelink/session/ISdlSessionListener.java

```diff
@@ -82,4 +83,6 @@
      * @param sessionID the session ID that this auth token is associated with
      */
     void onAuthTokenReceived(String authToken, int sessionID);
+
+    boolean onVehicleTypeReceived(VehicleType type, String systemSoftwareVersion, String systemHardwareVersion);
```

Comment:
@kostyaBoss Please update the file `MockInterfaceBroker.java` to include `onVehicleTypeReceived(...)`.  It doesn't build.